### PR TITLE
bpo-37035: Don't log OSError

### PR DESCRIPTION
**Pull request by asvetlov**
https://bugs.python.org/issue37035

### CODE REVIEW

**Comment by asvetlov** [comment]
ready for review

File: Misc/NEWS.d/next/Library/2019-05-24-18-16-07.bpo-37035.HFbJVT.rst [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Don't log OSError based exceptions if a fatal error is occured in asyncio
```
Comment (Carreau) [inline Misc/NEWS.d/next/Library/2019-05-24-18-16-07.bpo-37035.HFbJVT.rst]:
```suggestion
Don't log OSError based exceptions if a fatal error has occurred in asyncio
```

**Review by Carreau — APPROVED** [review]

File: Misc/NEWS.d/next/Library/2019-05-24-18-16-07.bpo-37035.HFbJVT.rst (hunk 1 above)
Comment (asvetlov) [reply Misc/NEWS.d/next/Library/2019-05-24-18-16-07.bpo-37035.HFbJVT.rst]: Thanks. English is not my strong point.

Comment (asvetlov) [reply Misc/NEWS.d/next/Library/2019-05-24-18-16-07.bpo-37035.HFbJVT.rst]: Thanks. English is not my strong point.

**Review by auvipy — APPROVED** [review]

File: Misc/NEWS.d/next/Library/2019-05-24-18-16-07.bpo-37035.HFbJVT.rst [hunk 2]
```diff
@@ -0,0 +1,2 @@
+Don't log OSError based exceptions if a fatal error has occurred in asyncio
```
Comment (1st1) [inline Misc/NEWS.d/next/Library/2019-05-24-18-16-07.bpo-37035.HFbJVT.rst:1]: Please expand this to include the motivation, e.g. that the errors will still propagate to the user code, it's just logging them is a bit pointless.

**Review by 1st1 — APPROVED** [review]
Please consider fixing the NEWS entry as I suggested

**Comment by asvetlov** [comment]
I've updated NEWS.
Is it ok now?

**Comment by 1st1** [comment]
> Is it ok now?

Yeah, thanks. This will make the what's new editor life easier.

**Comment by 1st1** [comment]
@asvetlov You sure it's a good idea to backport this to 3.7?  I wouldn't do that in a point release.

**Comment by asvetlov** [comment]
It is just logging.
No behavior changing.
If you strongly disagree I'll skip the backporting.
My motivation is that: now Python 3.7 users have their logs spammed with annoying messages. Applying this PR to 3.7 can help them a lot.
Now the only recommendation is `loop.set_exception_handler(lambda loop, ctx: None)` which is terrible.

**Comment by 1st1** [comment]
> Applying this PR to 3.7 can help them a lot.

OK; if you think so.

**Pull request merged**